### PR TITLE
chore(benchmark): add folder to ESLint

### PR DESCRIPTION
**Pull request by byzyk**
**What kind of change does this PR introduce?**

refactor

**Did you add tests for your changes?**

N/A

**Does this PR introduce a breaking change?**

No

**What needs to be documented once your changes are merged?**

N/A

### CODE REVIEW

File: benchmark/createTestCases.js [hunk 1]
```diff
@@ -50,20 +50,34 @@ function createTree(fs, count, folder) {
...
 				} else {
-					list.push(`import module${i} from "./${prefix}-${i}";\ncounter += module${i};`);
+					list.push(
+						`import module${i} from "./${prefix}-${i}";\ncounter += module${i};`
+					);
 				}
-				make(`${prefix}-${i}`, depth > 4 || count > 30 ? 0 : count + depth + i ** 2, depth + 1);
+				make(
+					`${prefix}-${i}`,
+					// eslint-disable-next-line node/no-unsupported-features
```
Comment (ooflorent) [inline benchmark/createTestCases.js]: Why is it needed? Could you remove this and fix the error?

File: benchmark/createFixtures.js [hunk 2]
```diff
@@ -5,55 +5,55 @@ const fixtures = path.join(__dirname, "fixtures");
 
 try {
 	fs.mkdirSync(fixtures);
-} catch(e) {}
+} catch (e) {
+	// catch here
```
Comment (ooflorent) [inline benchmark/createFixtures.js]: Could you change this to:

```js
// The directory already exists
```

File: benchmark/createTestCases.js (hunk 1 above)
Comment (byzyk) [reply benchmark/createTestCases.js]: So far I recall the `**` operator is not supported in our target version of Node. I'll have a look into it

File: benchmark/createBenchmark.js [hunk 3]
```diff
@@ -1,22 +1,29 @@
+// eslint-disable-next-line node/no-extraneous-require
```
Comment (byzyk) [inline benchmark/createBenchmark.js]: @ooflorent shall we import webpack relatively here same as we do in tests to eliminate this issue as well?

File: benchmark/createTestCases.js (hunk 1 above)
Comment (ooflorent) [reply benchmark/createTestCases.js]: You are right. Could you switch to `Math.pow`?

File: benchmark/createBenchmark.js (hunk 3 above)
Comment (ooflorent) [reply benchmark/createBenchmark.js]: Go ahead!

File: benchmark/createTestCases.js (hunk 1 above)
Comment (byzyk) [reply benchmark/createTestCases.js]: Yeah, I was thinking of the same, that should work 👍

**Comment by byzyk** [comment]
Is there a way to restart Appveyor build since it failed because of some weird jest timeout?

**Comment by sokra** [comment]
Thanks

**Pull request merged**